src/main/java/me/moodcat/backend/rooms/RoomInstance.java

```diff
@@ -182,6 +188,37 @@ protected void playNext(final Song song) {
 
         hasChanged.set(true);
     }
+    
+    private void processVotes(final Song previousSong) {
+        AtomicInteger nettoVotes = new AtomicInteger();
```

Comment:
If the integer cannot be accessed from multiple threads, there is no need to make it atomic.